src/pages/product-page/product-page.tsx

```diff
@@ -8,6 +8,9 @@ import { WixAPIContext } from '../../api/WixAPIContextProvider';
 import commonStyles from '../../styles/common-styles.module.scss';
 import { ProductImages } from './product-images/product-images';
 import { ProductInfo } from './product-info/product-info';
+import ProductInfo_module from './product-info/product-info.module.scss';
```

Comment:
why do we import all these styles?

Reply:
Didn't clean the files yet. Probabley from old project..

Reply:
stop using them and delete